rcp/org.testeditor.rcp4.tcltestrun/src/org/testeditor/rcp4/tcltestrun/TclMavenLauncher.xtend

```diff
@@ -38,22 +42,26 @@ public class TclMavenLauncher implements TclLauncher {
 	@Inject extension ProjectUtils
 	@Inject MavenExecutor mavenExecutor
 
-	override LaunchResult launchTest(List<String> testCases, IProject project, IProgressMonitor monitor, OutputStream out,
-		Map<String, Object> options) {
+	override LaunchResult launchTest(List<String> testCases, IProject project, IProgressMonitor monitor,
+		OutputStream out, Map<String, Object> options) {
 		val parameters = if (options.containsKey(
 				PROFILE)) {
 				"clean generate-test-sources org.testeditor:testeditor-maven-plugin:testEnvUp org.testeditor:testeditor-maven-plugin:testExec -P" +
 					options.get(PROFILE)
 			} else {
 				"clean integration-test"
 			}
-			
+
 		var testSelectionArgument = ""
-		if(testCases != null) {
+		if (testCases != null) {
 			testSelectionArgument = "test=" + testCases.join(",")
-		} 
-		val result = mavenExecutor.executeInNewJvm(parameters, project.location.toOSString,
-			testSelectionArgument, monitor, out, false)
+		}
+		if(!checkMavenVersion(project, monitor)) {
+			return new LaunchResult(null, false, null)			
+		}
+
+		val result = mavenExecutor.executeInNewJvm(parameters, project.location.toOSString, testSelectionArgument,
```

Comment:
This is my analysis of the code: There is the parameter useJvmClasspath of the method createMavenExecCommand. This variable has to be set to true to use the external maven. But the variable seemed to be set to false by default and I did not find any code to set it to true. If this is correct, the embedded maven is always used? By the way the embedded maven has the version 3.3.9.